cloudinit/distros/networking.py

```diff
@@ -214,3 +221,10 @@ def settle(self, *, exists=None) -> None:
         if exists is not None:
             exists = net.sys_dev_path(exists)
         util.udevadm_settle(exists=exists)
+
+    def try_set_link_up(self, devname: DeviceName) -> bool:
```

Comment:
We should add tests for this to [`test_networking.py`](https://github.com/canonical/cloud-init/blob/master/cloudinit/distros/tests/test_networking.py).